rocAL/rocAL/source/meta_data/meta_node_resize_crop_mirror.cpp

```diff
@@ -67,31 +71,35 @@ void ResizeCropMirrorMetaNode::update_parameters(pMetaDataBatch input_meta_data,
         crop_box.b = _y2_val[i];
         for(uint j = 0; j < bb_count; j++)
         {
+            
             if (BBoxIntersectionOverUnion(box_coords_buf[j], crop_box) >= _iou_threshold)
             {
                 float xA = std::max(crop_box.l, box_coords_buf[j].l);
                 float yA = std::max(crop_box.t, box_coords_buf[j].t);
                 float xB = std::min(crop_box.r, box_coords_buf[j].r);
                 float yB = std::min(crop_box.b, box_coords_buf[j].b);
-                box_coords_buf[j].l = (xA - crop_box.l) / (crop_box.r - crop_box.l);
-                box_coords_buf[j].t = (yA - crop_box.t) / (crop_box.b - crop_box.t);
-                box_coords_buf[j].r = (xB - crop_box.l) / (crop_box.r - crop_box.l);
-                box_coords_buf[j].b = (yB - crop_box.t) / (crop_box.b - crop_box.t);
+                box_coords_buf[j].l = (xA - crop_box.l);
```

Comment:
Why are we not dividing here?

Reply:
The previous implementation was normalizing the output coords with the crop box width and height. I changed that and Akilesh verified the outputs for this change with ToT and made sure the values were the same numerically and visually. I'll ask him to verify once more

Reply:
Check the formula once. You need to verify it with SSD / DALI.